CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - More detailed and sophisticated search space user guide
 - Upgraded syntax to Python 3.9
+- Use `tox-uv` in pipelines
```

Comment:
any reason were not also changing the dev dependencies in pyproject.toml?
imo locally we should then also use tox-uv

Reply:
yes, good point. Let me first check if it works anyway. If everything's OK, I'll also change it in the toml

Reply:
Looking good! I also added the timings from my local machine to the PR message. Will now also compare if makes a difference in CI, but the PR itself is ready for merge I think

Reply:
ty